dp_consistency/src/correctoriginreport.py

```diff
@@ -0,0 +1,164 @@
+from src.noiseninja.noised_measurements import Measurement
+from src.report.report import Report, MetricReport
+import sys
+import argparse
+import pandas as pd
+from functools import partial
+
+# This is a demo script that has the following assumptions :
+#   1. There are 2 EDPs one with Name Google, the other Linear TV.
+#   2. CUSTOM filter is ignored since it won't be used for the forseeable future.
+#   3. AMI is a parent of MRC and there are no other relationships between metrics.
+#   4. The variance is assumed to be 1??
+#   5. Frequency results are not corrected.
+#   6. Impression results are not corrected.
+#   7. The issue with the Linear TV shouldn't be corrected can be addressed by setting the variance of its measurements to 0.
+
+
+# Important Note: If sigma is set to a low number like 1, the corrected report returns numbers very close to 0. 
+SIGMA = 100
+
+AMI_FILTER = "AMI"
+MRC_FILTER = "MRC"
+
+EDP_ONE = "Google"
```

Comment:
Let's add a TODO to read EDP names dynamically from the file, and to make this work for more than 2 EDPs.

Reply:
done